src/modules/analysis/AnalysisEditComponent.js

```diff
@@ -30,7 +30,7 @@ type ObjectData = { uuid: string };
 
 type Store = {
   objectsData: ObjectData[],
-  analysisTypes: AnalysisType[]
+  analysisTypes: [AnalysisType]
```

Comment:
I prefer that we use the more explicit way of declaring arrays. 

```
analysisTypes: Array<AnalysisType>
```

By doing the change here we're having three ways of doing the same thing.